modules/moa-accessing-your-cluster.adoc

```diff
@@ -10,31 +10,58 @@ To log in to your cluster, you must configure an Identity Provider (IDP).
 
 This procedure uses GitHub as an example IDP. 
 
+To view other supported IDPs, run `moactl create idp --help`.
```

Comment:
Not critical, but I think that 3 separate lines for 3 sentences looks a little odd. I'd consider moving this sentence up to after "This procedure uses GitHub as an example IDP.", since it seems to be related.